app/views/posts/index.html.erb

```diff
@@ -1,12 +1,62 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Posts</title>
-  </head>
-  <body>
-    <h1>list of the posts shown for a designated user.</h1>
-  </body>
-</html>
\ No newline at end of file
+<div class="sub-container" >
+
+  <!-- User Profile -->
+  <div class="card mb-3">
+    <div class="row g-0">
+      <div class="col-md-4">
+        <img src="<%= @user.photo %>" class="img-fluid img-thumbnail rounded-start" alt="<%= @user.name %>">
+      </div>
+      <div class="col-md-8">
+        <div class="card-body">
+          <h5 class="card-title"><%= @user.name %></h5>
+          <p class="card-text text-end">Number of posts: <%= @user.posts_counter || 0 %> </p>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- User Bio -->
+  <div class="card mb-2">
+    <div class="card-body">
+      <h6 class="card-title">Bio</h6>
+      <p class="card-text"><%= @user.bio %></p>
```

Comment:
- [ ] [OPTIONAL AND IMPORTANT] As you can see this user has created only one post but the number of posts says three, I figured that it counts both comments and likes as posts, kindly fix that
![Screenshot from 2023-11-10 19-20-19](https://github.com/ethred/blog_app/assets/66097371/6a6a292c-d2cd-48dd-a7ce-8fb345d93266)